phoenix-core/src/main/java/org/apache/phoenix/coprocessor/metrics/MetricsPhoenixCoprocessorSourceFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.apache.phoenix.coprocessor.metrics;
+
+public class MetricsPhoenixCoprocessorSourceFactory {
+
+    private static final MetricsPhoenixCoprocessorSourceFactory
```

Comment:
Can we do lazy initialization instead?

Reply:
Not sure if there is any value to lazy initialization as it is a simple object creation. Not creating any expensive resources.